source/public/scripts/main.journal.js

```diff
@@ -0,0 +1,127 @@
+document.addEventListener("DOMContentLoaded", function () {
+    // Select the modal and various buttons
+    var modal = document.getElementById("task-modal");
```

Comment:
`var` is kind of an outdated JavaScript variable declaration, since these elements aren't being reassigned it would be better to use `const` instead

Reply:
Also if you could use a bit more descriptive names for the variables like `taskModal` instead of just `modal` that would be helpful for code readbility!